.github/workflows/ci.yml

```diff
@@ -150,6 +150,9 @@ jobs:
           export IMAGE_PROCESSING_CONTAINER_IMAGE="$(KO_DOCKER_REPO=kind.local ko publish ./cmd/image-processing)"
 
           make test-integration
+      - name: Test-PipelineRun
+        run: |
+          BUILDRUN_EXECUTOR=PipelineRun ginkgo --focus-file="buildruns_to_pipelineruns_test.go" -v test/integration/...
```

Comment:
nit (not blocking): ideally would like to have these be arguments to the make target, but I can live with this for now.